src/vscode-dts/vscode.d.ts

```diff
@@ -20293,6 +20303,62 @@ declare module 'vscode' {
 		constructor(label: string, command: string, args?: string[], env?: Record<string, string | number | null>, version?: string);
 	}
 
+	/**
+	 * McpPackageServerDefinition represents an MCP server available by running
+	 * a local process with the provided package and operating on its stdin and stdout streams.
+	 * The process will be spawned as a child process of the extension host and by default
+	 * will not run in a shell environment.
+	 */
+	export class McpPackageServerDefinition {
+		/**
+		 * The human-readable name of the server.
+		 */
+		readonly label: string;
+
+		/**
+		 * Name of the package that needs to be started.
+		 */
+		readonly packageName: string;
+
+		/**
+		 * The type of the mcp package
+		 */
+		readonly packageType: McpPackageType;
+
+		/**
+		 * The working directory used to start the server.
+		 */
+		cwd?: Uri;
+
+		/**
+		 * Additional command-line arguments passed to the server.
+		 */
+		args: string[];
+
+		/**
+		 * Optional additional environment information for the server. Variables
+		 * in this environment will overwrite or remove (if null) the default
+		 * environment variables of the editor's extension host.
+		 */
+		env: Record<string, string | number | null>;
+
+		/**
+		 * Optional version identification for the server. If this changes, the
+		 * editor will indicate that tools have changed and prompt to refresh them.
+		 */
+		version?: string;
```

Comment:
Should this be required in the package definition? We generally don't want to just point at `@latest` since that can break unexpectedly.